components/user_settings/general/user_settings_general.tsx

```diff
@@ -304,7 +303,11 @@ export class UserSettingsGeneralTab extends React.Component<Props, State> {
             then(({data, error: err}) => {
                 if (data) {
                     this.updateSection('');
-                    this.props.actions.getMe();
+
+                    // this is handled by actions/websocket_actions/handleUserUpdatedEvent
+                    // ref: https://github.com/mattermost/mattermost-webapp/pull/9724
+                    // this.props.actions.getMe();
```

Comment:
I think we can actually remove this comment. This is actually handled by [the `updateMe` action](https://github.com/mattermost/mattermost-webapp/blob/master/packages/mattermost-redux/src/actions/users.ts#L1078), so I don't know why we would've done it here as well. I have to assume it goes back to some really old version of the code

Reply:
I found the source of it. Back before Redux in the initial public version of Mattermost, we called the API directly to update the user, and then we pulled the user from the API again instead of reading the response for some reason. No idea why that happened, but it's lasted this long in the code in some form or another.

https://github.com/mattermost/mattermost-server/blob/56e74239d6b34df8f30ef046f0b0ff4ff0866a71/web/react/components/user_settings.jsx#L758

Reply:
Understood, I've removed this comment.